tests/core/pyspec/eth2spec/test/helpers/altair/fork.py

```diff
@@ -0,0 +1,42 @@
+ALTAIR_FORK_TEST_META_TAGS = {
+    'fork': 'altair',
+}
+
+
+def run_fork_test(post_spec, pre_state):
+    # Clean up state to be more realistic
+    pre_state.current_epoch_attestations = []
```

Comment:
why clear these? on mainnet at fork time i would expect there to be data in here...

Reply:
Because `process_participation_record_updates` was just executed and cleared `current_epoch_attestations`?

@djrtwo, and  this line makes `state_0.current_epoch_attestations = state_0.current_epoch_attestations + state_1.current_epoch_attestations` in `test_altair_fork_random_mismatched_attestations` (L90 in `test_random_altair_fork.py`) meaningless. 😅  Maybe remove `state_0.current_epoch_attestations = state_0.current_epoch_attestations + state_1.current_epoch_attestations`?

Reply:
ah true!

Reply:
cleaned up!